app-ui/package.json

```diff
@@ -25,6 +25,8 @@
     "@ng-web-apis/common": "^3.0.6",
     "@ng-web-apis/speech": "^3.0.7",
     "@ngneat/until-destroy": "^8.1.4",
+    "file-saver": "^2.0.5",
+    "fs": "^0.0.1-security",
```

Comment:
Malicious package:

Node js used to have fs package before but they have renamed it to fs-extra, someone took it as oppurtunity and created this

https://www.npmjs.com/package/fs?activeTab=readme